### PR TITLE
docs(icons.md): update nonsensical language

### PR DESCRIPTION
**Pull request by rochdiNassah**

### CODE REVIEW

**Review by jacekkarczmarczyk — DISMISSED** [review]

**Comment by KaelWD** [comment]
That doesn't make any sense either, is it supposed to say "throw on a `role="img"` attribute"?

"The attribute and the element are added" is nonsense too.

**Comment by KaelWD** [comment]
> you need to provide a text alternative inside a (or similar) element

This whole page is cooked, who wrote it?

**Comment by johnleider** [comment]
I hate this entire sentence as it stands. Let's either rewrite what it says or close this PR.

File: packages/docs/src/pages/en/components/icons.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ If your icons are purely decorative, you’ll need to manually add an attribute
 
 ### Semantic SVG Icons
 
-If your icon has semantic meaning, all you need to do is throw a attribute.The attribute and the element are added so that your icons are properly accessible.`role="img"<v-icon>`.
+If your icon has semantic meaning, all you need to do is throw an attribute.The attribute and the element are added so that your icons are properly accessible.`role="img"<v-icon>`.
```
Comment (johnleider) [inline packages/docs/src/pages/en/components/icons.md:117]:
```suggestion
Apply accessibility attributes to the [v-icon](/components/icons/), such as `role="img"`, to the component to give it a semantic meaning.
```

**Review by johnleider — CHANGES_REQUESTED** [review]

File: packages/docs/src/pages/en/components/icons.md [hunk 2]
```diff
@@ -114,7 +114,7 @@ If your icons are purely decorative, you’ll need to manually add an attribute
 
 ### Semantic SVG Icons
 
-If your icon has semantic meaning, all you need to do is throw a attribute.The attribute and the element are added so that your icons are properly accessible.`role="img"<v-icon>`.
+Apply accessibility attributes to the [v-icon](/components/icons/), such as `role="img"`, to the component to give it a semantic meaning.
```
Comment (johnleider) [inline packages/docs/src/pages/en/components/icons.md:117]:
```suggestion
Apply accessibility attributes to the [v-icon](/components/icons/) component, such as `role="img"`, to give it a semantic meaning.
```

**Review by johnleider — APPROVED** [review]

**Pull request merged**